src/Tickets/Commerce/Gateways/PayPal/Client.php

```diff
@@ -131,19 +137,52 @@ public function get( $endpoint, array $query_args = [], array $request_arguments
 				'Content-Type'  => 'application/json',
 			]
 		];
+
+		if ( 'GET' !== $method ) {
+			$default_arguments['body'] = [];
+		}
```

Comment:
We're having the same conditional in the same function with a few lines difference. I assume this is to initialize the $default_arguments or something. Can we possibly leave some inline documentation to justify that we're doing this so that's not removed by some other dev in the future?

Reply:
WIll make sure to add some comments, I like the idea.